### PR TITLE
add language if posting multiple roles

### PR DESCRIPTION
**Pull request by wslack**

### CODE REVIEW

File: pages/phases/job-announcement/create-job-announcement.md [hunk 1]
```diff
@@ -37,6 +37,10 @@ To ensure your hiring process will be manageable for participating SMEs, clearly
 
 Duties should be informed by the job tasks you documented during job analysis. Place the most important job duties as the first few bullets.
 
+If you are posting for multiple roles (this is not typical), and your SMEs determine in job analysis that applicants can only reasonably apply for a subset of roles, include this text in this section (moddifying the specific number as appropriate):
```
Comment (stephaniefaith) [inline pages/phases/job-announcement/create-job-announcement.md:40]:
```suggestion
If you are posting for multiple roles (this is not typical), and your SMEs determine in job analysis that applicants should only reasonably be allowed to apply for a subset of those roles, include this text in this section (modifying the specific number as appropriate):
```

**Review by stephaniefaith — COMMENTED** [review]
can you confirm Ro is okay with us advertising this? if she is, maybe we need to explain that the talent acquisition system can't limit the number of selections so you need to tell applicants if they choose more than x number, we'll only consider them for x number even if they select all 4. should we also go over the motivation to prevent serial appliers?

**Comment by wslack** [comment]
will do

**Comment by wslack** [comment]
Ro confirmed; this is fine if the SMEs endorse.

**Review by stephaniefaith — APPROVED** [review]

**Pull request merged**